java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -2876,6 +2876,23 @@ void testListConcatByRowIgnoreNull() {
     }
   }
 
+  @Test
+  void testFlattenLists() {
+    HostColumnVector.ListType listType = new HostColumnVector.ListType(true,
+        new HostColumnVector.BasicType(true, DType.INT32));
+    HostColumnVector.ListType listOfListsType = new HostColumnVector.ListType(true, listType);
+
+    try (ColumnVector input = ColumnVector.fromLists(listOfListsType,
+             Arrays.asList(Arrays.asList(1, 2), Arrays.asList(3), Arrays.asList(4, 5, 6)),
+             Arrays.asList(Arrays.asList(7, 8, 9), Arrays.asList(10, 11, 12, 13, 14, 15)));
+         ColumnVector result = input.flattenLists();
```

Comment:
do we have a test for flattenLists(true) - to ignoreNulls ?

Reply:
Sure we can have such test.